src/plugins/walletConnectToDapps/eventsManager/useWalletConnectEventsHandler.ts

```diff
@@ -92,11 +97,45 @@ export const useWalletConnectEventsHandler = (
 
         case EIP155_SigningMethod.WALLET_ADD_ETHEREUM_CHAIN:
         case EIP155_SigningMethod.WALLET_SWITCH_ETHEREUM_CHAIN:
-          // This doesn't fit our usual pattern and doesn't require a modal
-          // We implicitly return a happy JSON-RPC response
```

Comment:
Welp that didn't age nicely

<img width="1000" height="188" alt="Image" src="https://github.com/user-attachments/assets/5081fa71-114a-49fe-ba59-3f0932ef4925" />